.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "cSpell.words": [
+        "closebrackets",
+        "codemirror",
+        "csstype",
+        "Keymap",
+        "matchbrackets"
+    ]
+}
```

Comment:
Please remove this file from git

Reply:
@SamuelAlev said that the settings.json should not be ignored. I initially put it to ignore as it was empty, but I agree that settings.json should not be ignored. In my mind, Code Spell Checker is something we should introduce anyway, and it does not harm anyone who is not using it.